lib/package.json

```diff
@@ -27,17 +28,19 @@
     "prepublishOnly": "npm run build && cp ../README.md ./"
   },
   "dependencies": {
-    "@storyblok/js": "^1.8.6",
+    "@storyblok/js": "^2.0.1",
```

Comment:
I think the RichTextSchema should be back from version "^2.0.4"

Reply:
thank you :-)